pact-tests/pact-tests/yield.repl

```diff
@@ -92,7 +92,7 @@
 
 ;; check events
 (expect "step 0 events"
- [{"module-hash": "oKFbzkuEYAFhP-S2mW7hRvRJUPJf2FvMFy1CpxhGs4o"
+ [{"module-hash": "rmN99MpBmJbapgVRV3GjII6I4UUkl8k5pBF7-k92Lt8"
```

Comment:
isn't the module-hash based on stable encoding? It shouldn't change?

Reply:
It's not based on stable encoding. It _should_ change. It's based on our CBOR encoding.